__tests__/app.test.js

```diff
@@ -113,3 +113,36 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/:article_id/comments", () => {
+  test("Should return 200 and relevant array of comments when given valid id", () => {
```

Comment:
we should also have a test that checks what happens when a valid article id is provided but the article has no comments